init.go

```diff
@@ -0,0 +1,97 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package dd_sdk_go_testing
+
+import (
+	"context"
+	"fmt"
+	"github.com/DataDog/dd-sdk-go-testing/internal/constants"
+	"github.com/DataDog/dd-sdk-go-testing/internal/utils"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
```

Comment:
nit: separate these from standar imports?

Reply:
oh, right I miss this file.

Reply:
done in 764a371